server/controllers/app.js

```diff
@@ -6,8 +6,15 @@ const bodyParser = require('body-parser');
 const Product = require('../models/Product');
 
 const app = express();
+app.all('/*', function (req, res, next) {
```

Comment:
I'm just curious, was this necessary for your server to run or did you just add it for good practice.

Reply:
Currently it's necessary as I need to get data from mongo. But could also hard code data for now, to reduce complexity?